apps/zero-runtime-vite-app/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "allowImportingTsExtensions": true
```

Comment:
Need to explicitly do `import Demos from '../path/to/Demo.tsx` to use tsx file because `.js` is not out of the box supported in Vite.